src/diffusers/schedulers/scheduling_dpmsolver_singlestep.py

```diff
@@ -108,9 +108,7 @@ class DPMSolverSinglestepScheduler(SchedulerMixin, ConfigMixin):
             The threshold value for dynamic thresholding. Valid only when `thresholding=True` and
             `algorithm_type="dpmsolver++"`.
         algorithm_type (`str`, defaults to `dpmsolver++`):
-            Algorithm type for the solver; can be `dpmsolver`, `dpmsolver++`, `sde-dpmsolver` or `sde-dpmsolver++`. The
-            `dpmsolver` type implements the algorithms in the [DPMSolver](https://huggingface.co/papers/2206.00927)
-            paper, and the `dpmsolver++` type implements the algorithms in the
+            Algorithm type for the solver; can be `dpmsolver++` or `sde-dpmsolver++`. The `dpmsolver++` type implements the algorithms in the
```

Comment:
`sde-dpmsolver++` is not supported whereas the doc states it is supported. I guess this documentation should be updated.

Reply:
yes will update the doc now. I Will try to add the support for sde-dpmsolver++ soon too. It's on my to-do for a long time now